docs/README.md

```diff
@@ -1,28 +1,152 @@
 # Crossplane
 
-Crossplane is an open source multicloud control plane. It introduces workload and resource abstractions on-top of existing managed services that enables a high degree of workload portability across cloud providers. A single crossplane enables the provisioning and full-lifecycle management of services and infrastructure across a wide range of providers, offerings, vendors, regions, and clusters. Crossplane offers a universal API for cloud computing, a workload scheduler, and a set of smart controllers that can automate work across clouds.
+Crossplane is an open source multicloud control plane to manage your
+cloud-native applications and infrastructure across environments, clusters,
+regions and clouds. It enables deployment of applications and managed services
+from `kubectl` with your choice of cloud or on-premise providers, including
+independent service offerings. 
```

Comment:
Could this be worded to more clearly represent the fact that in some cases this is our goal and not necessarily our current state?

It's true that we _enable_ on-prem and independent service offerings, in that we provide a framework to build those things, but at the time of writing no on-prem or independent infrastructure stacks exist. If I'm a potential Crossplane OSS user this is going to be one of the first things I read. This wording would make me assume the functionality exists today, dig around a bit, and pretty quickly discover it does not. Speaking personally, I've encountered this pattern in which OSS claims to do a thing that it does not yet do (especially lately in the cloud native ecosystem), and it typically lowers my opinion of those projects quite a lot.

Reply:
Updates LGTM.